### PR TITLE
Upgrade sentry SDK

### PR DESCRIPTION
**Pull request by piotrpekala7**

### CODE REVIEW

File: src/app/common/error-handlers/toaster-error-handler.ts [hunk 1]
```diff
@@ -1,7 +1,8 @@
 import { RavenErrorHandler } from './raven-error-handler';
```
Comment (grossmj) [inline src/app/common/error-handlers/toaster-error-handler.ts:1]: I think you forgot to remove the RavenErrorHandler.

File: package.json [hunk 2]
```diff
@@ -49,6 +49,7 @@
     "@angular/platform-browser": "^8.2.11",
     "@angular/platform-browser-dynamic": "^8.2.11",
     "@angular/router": "^8.2.11",
+    "@sentry/browser": "^5.18.0",
```
Comment (grossmj) [inline package.json:52]: Remove the `raven-js` dependency.

File: src/app/common/error-handlers/toaster-error-handler.ts (hunk 1 above)
Comment (piotrpekala7) [reply src/app/common/error-handlers/toaster-error-handler.ts]: fixed

File: package.json (hunk 2 above)
Comment (piotrpekala7) [reply package.json:52]: fixed

**Comment by grossmj** [comment]
Have you tested this work correctly? Try with a division by zero somewhere in the code and check on Sentry that the error has been received.

**Comment by grossmj** [comment]
@piotrpekala7 ping

**Comment by piotrpekala7** [comment]
crashReport

works as expected

**Comment by grossmj** [comment]
Interesting, I didn't know Sentry can request more information from users. Looks great, thanks :+1:

**Pull request merged**